socialhome/streams/app/components/StreamElement.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+    <div>
+        <p v-html="post.content"></p>
+        <div class="grid-item-author-bar mt-1">
+            <div class="profile-box-trigger">
+                <img :src="post.author.imageUrlSmall" class="grid-item-author-bar-pic">
+            </div>
+            <div class="profile-box hidden">
+                {{ post.author.handle }}
+                <div class="pull-right">
+                    <a :href="post.author.absoluteUrl" class="btn btn-secondary"
```

Comment:
Multi-line tags look less confusing if the closing tag is on a brand new line like you've done in many other places. Also the `i` indentation should be one space more.